x-pack/metricbeat/module/aws/_meta/docs.asciidoc

```diff
@@ -37,6 +37,15 @@ or none get collected by Metricbeat. In this case, please specify a `latency`
 parameter so collection start time and end time will be shifted by the given
 latency amount.
 
+* *data_granularity*
+
+This module accepts an optional `data_granularity` parameter which sets the granularity of the returned data points.
+This parameter can be used to decouple how frequently metricbeat would collect data from AWS, from how granular this data should be.
+This would helpful to reduce the cost incurred in calling AWS Cloudwatch, in cases where highly-granular data is required,
+but a delay in collecting and ingesting the metrics can be tolerated.
+
+If no value is specified, the value is set to the `period`.
```

Comment:
Can you add somewhere here some "problems - approach" to give some sense of scope? For example: "if you see that you Cloudwatch API costs (seen in xxx) is too high, increase / decrease this parameter to Y value. A good starting point to reduce costs is Z and then start experimenting from there".

No need to use that exact wording, but I think you get the idea 😉 Thanks!

Reply:
Thanks for reviewing this. I thought to keep this part brief  as we will probably revamp the AWS docs either way, but yours is a totally valid point. We could re-use the scenarios explained https://github.com/elastic/integrations/issues/2913#issuecomment-1242266617 and summarize them. 

For @legoguy1000, if not clear, there is no expectations from your side to make these doc changes ;)